examples/geoscience-aus-cat/projects_geoscience-aus-cat_assets_ga_ls8cls9c_gm_cyear_3_preview.js

```diff
@@ -0,0 +1,39 @@
+var geomedian_ls8ls9 = ee.ImageCollection('projects/geoscience-aus-cat/assets/ga_ls8cls9c_gm_cyear_3');
+
+var geometry = 
+    /* color: #98ff00 */
+    /* displayProperties: [
+      {
+        "type": "rectangle"
+      }
+    ] */
+    ee.Geometry.Polygon(
+      [[[121.15880998755823, -15.010654451073695],
+        [121.15880998755823, -18.377531570740548],
+        [125.81701311255823, -18.377531570740548],
+        [125.81701311255823, -15.010654451073695]]], null, false);
+
+// Width and height of the thumbnail image.
+var pixels = 256;
+      
+var composite = geomedian_ls8ls9.filterBounds(geometry)
+                             .filterDate('2018-01-01', '2019-01-01')
+                             .mosaic();
+                             
+var visualization = {
+  bands: ['nbart_red', 'nbart_green', 'nbart_blue'],
+  min: 0,
+  max: 3000
+};
+
+Map.centerObject(geometry, 10);
+Map.addLayer(composite, visualization, '2018 True Color Composite');
+
+var imageParams = {
+  dimensions: [pixels, pixels],
+  region: geometry,
+  crs: 'EPSG:3857',
+  format: 'png',
+};
+
+print(ui.Thumbnail({image: composite.visualize(visualization), params: imageParams}));
```

Comment:
This thumbnail is partially masked out, so it won't look like a neat square. Could you either shift it to an area that's not fully masked out or use gray background like in https://github.com/google/earthengine-catalog/blob/main/examples/WM/WM_geoLab_geoBoundaries_600_ADM0_preview.js#L21 ?

Reply:
I changed the thumbnail area, now it should show a neat square.